package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Deferral poject",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon --exec babel-node index.js",
+    "start": "node build/index.js",
```

Comment:
I was wondering why you previously had nodemon in the start script. Looks cool now.
Or, was it there for a reason?

Reply:
None bro. It was an error IG